rest-api/src/main/java/gov/cms/qpp/conversion/api/helper/MetadataHelper.java

```diff
@@ -10,12 +11,15 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
+import java.util.Random;
 
 /**
  * Utilities for working with Metadata beans
  */
 public class MetadataHelper {
 
+	private static final Random RANDOM_HASH = new Random();
```

Comment:
small note about sharing instances of Random that shouldn't really be an issue for this application, but it's still worth thinking about -- while Random is thread safe, sharing an instance in a place of high contention is generally a bad idea & decreases throughput.

Also, is this somewhere that SecureRandom should be used instead?

Reply:
`SecureRandom` shouldn't be needed here.  We are not doing a cryptographic operation.
Good point on the contention.  I'll change it to a `ThreadLocalRandom`.